python/lsst/meas/algorithms/psfCandidate/psfCandidateContinued.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+from lsst.utils import continueClass
+from .psfCandidate import PsfCandidateF
+
+__all__ = []  # import only for side effects
```

Comment:
I'm sure I've missed instances of this, too, but as @r-owen recently pointed out on the meas_modelfit review, we should be putting `__all__` above the imports (but below `from __future__`) according to PEP8.